.vsts-ci.yml

```diff
@@ -58,275 +48,300 @@ stages:
         name: NetCore1ESPool-Internal
         demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre
     steps:
-    - publish: $(Build.SourcesDirectory)\eng\BuildConfiguration
-      artifact: BuildConfiguration
+    - publish: $(Build.SourcesDirectory)\eng\buildConfiguration
+      artifact: buildConfiguration
       displayName: Publish Build Config
-  - template: /eng/build.yml
-    parameters:
-      agentOs: Windows_NT
-      pool:
-        ${{ if eq(variables['System.TeamProject'], 'public') }}:
-          name: NetCore-Public
-          demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre.Open
-        ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-          name: NetCore1ESPool-Internal
-          demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre
-      timeoutInMinutes: 180
-      strategy:
-        matrix:
-          # Public-only builds
-          ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-            Build_Release_x64:
-              _BuildConfig: Debug
-              _BuildArchitecture: x64
-              _DOTNET_CLI_UI_LANGUAGE: ''
-              _AdditionalBuildParameters: '/p:PublishInternalAsset=true'
-              _TestArg: $(_WindowsTestArg)
-          # Internal-only builds
-          ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-            Build_Release_x64:
-              _BuildConfig: Release
-              _BuildArchitecture: x64
-              _DOTNET_CLI_UI_LANGUAGE: ''
-              _AdditionalBuildParameters: '/p:PublishInternalAsset=true'
-              _TestArg: $(_WindowsTestArg)
-            Build_Release_x86:
-              _BuildConfig: Release
-              _BuildArchitecture: x86
-              _DOTNET_CLI_UI_LANGUAGE: ''
-              _AdditionalBuildParameters: ''
-              _TestArg: $(_WindowsTestArg)
-            Build_Release_arm64:
-              _BuildConfig: Release
-              _BuildArchitecture: arm64
-              _DOTNET_CLI_UI_LANGUAGE: ''
-              _AdditionalBuildParameters: ''
-              # Never run tests on arm64
-              _TestArg: ''
+  
+  ## PR-only jobs
+  
+  - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+    
+    ## Windows
+
+    - template: /eng/build.yml
+      parameters:
+        agentOs: Windows_NT
+        jobName: Build_Debug_x64
+        buildConfiguration: Debug
+        buildArchitecture: x64
+        additionalBuildParameters: '/p:PublishInternalAsset=true'
+        runTests: true
+
+    ## Linux
+
+    - template: /eng/build.yml
+      parameters:
+        agentOs: Linux
+        jobName: Build_Ubuntu_18_04_Debug_x64
+        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-20220916154732-3c53da6'
```

Comment:
some of the Dockerfiles used previously were potentially installing additional dependencies. Have we made sure that the dockerfiles from builtools prereqs have those dependencies?

Reply:
Yep..I've run a ton of builds. I am thinking that most of the dependencies were not used.

Reply:
neat